app/workers/mails.php

```diff
@@ -35,9 +35,9 @@ public function run(): void
 
         $recipient = $this->args['recipient'];
         $subject = $this->args['subject'];
-        $name = $this->args['name'];
         $body = $this->args['body'];
         $variables = $this->args['variables'];
+        $name = $this->args['name'];
```

Comment:
```suggestion
        $name = $variables['user'] ?? '';
```

Reply:
This should `args['name']` as it is used as `from` name, `variables['user']` is the `to` name